config.json

```diff
@@ -1,4 +1,6 @@
 {
+  "appVersion": "3.2.0",
```

Comment:
Can you rather make this an optional override instead of putting it into the git config? Otherwise user deployments will need to resolve merge conflicts every time they update cinny - or plainly have a wrong app version in there if they just `cp old-cinny/config.json new-cinny/config.json` from the old deployment.